docs/source/models/gbd2017/causes/diarrheal_diseases.rst

```diff
@@ -61,10 +61,75 @@ I: _I_nfected and currently experiencing a diarrheal disease bout
 
 Data Descriptions
 -----------------
+	 
+	 
+.. list-table:: State definitions and initialization
+   :widths: 5 30
+   :header-rows: 1
+
+   * - State
+     - Definition
+   * - I
+     - Currently has diarrheal disease.
+   * - S
+     - Does not currently have diarrheal disease.
+	 
+.. csv-table:: Prevalence
+   :header: State,Prevalence,Source
+   :widths: 5, 10, 10
+   :stub-columns: 0
+
+   I,prev_302,como
+   S,1-prev_302,NA
+
+prev_302 = get_draws("cause_id", 302, source = "como", measure_id = 5, gbd_round_id = 5)
+
+.. csv-table:: Transitions
+   :header: Transition,Rate (per person-year),Source, Note
+   :widths: 10, 20, 10, 30
+   :stub-columns: 0
+
+   I -> S,rem_302,epi,Already a rate within with-condition population
+   S -> I,inc_302/(1-prev_302),como,We transform incidence to be a rate within the susceptible population.
+
+inc_hazard_302 = get_draws("cause_id", 302, source = "como", measure_id = 6, gbd_round_id = 5)
+rem_302 = get_model_results(gbd_team='epi', gbd_id=1181, gbd_round_id=5, measure_id = 7)
+
+
+
+.. csv-table:: Excess Mortality Rate
+   :header: State,EMR,source
+   :widths: 5, 10, 10
+   :stub-columns: 0
+
+   I,(deaths_302/pop)/prev_302,codcorrect
+   S,0,NA
+
+deaths_302 = get_outputs("cause", cause_id = 302, measure_id = 1, metric_id = 1, gbd_round_id = 5)
+
+.. csv-table:: Disability Weights
+   :header: Severity level, DW (95% CI),Proportion of estimated diarrhea
+   :widths: 10, 20, 10
+   :stub-columns: 0
+
+   Mild,0.0074 (0.049-0.104),.648
+   Moderate,0.188 (0.125-0.264),.289
+   Severe,0.247 (0.164-0.348),.069
+
+Note, the above proportions sum to 1.006. These numbers come from the appendix
+model writeup [GBD-2017-YLD-Capstone-Appendix-1]_. The severity splits come from a
+meta-analysis on severity independent from the DisMod estimates. They splits are then
+applied to the prevalence and incidence estimates, in order to calculate YLDs by sequela and
+etiology.
+
+.. todo::
+
+	Look into what severity splits were actually used.
```

Comment:
Severity splits will vary by demography.  These are, to my knowledge, based strongly on healthcare utilization information (an important thing to note in the model limitations section when you get to it).

Reply:
Maybe add James's comment to the todo?

Reply:
Beatrix and I talked about this during coworking and I think it drops out here and just goes in the limitations section.